src/webrtc/peerconnection.ts

```diff
@@ -183,6 +189,10 @@ export class PeerConnectionClass implements PeerConnection<signals.Message> {
     this.fulfillClosed_ = F;
   });
 
+  private controlChannel_ :datachannel.ControlChannel = null;
```

Comment:
Suggest you instead define a `Promise` that resolves with the control channel. There's a bunch of other such promises here, e.g. this `void`-resolving example:
```typescript
  private fulfillConnected_ :() => void;
  private rejectConnected_ :(e:Error) => void;
  public onceConnected = new Promise<void>((F,R) => {
    this.fulfillConnected_ = F;
    this.rejectConnected_ = R;
  });
```